cluster/gce/util.sh

```diff
@@ -642,7 +646,11 @@ function kube-up {
     scope_flags=("--no-scopes")
   fi
 
-  write-node-env
+  if [[ "${TEST_NODE:-}" == "true" ]]; then
```

Comment:
The problem that remains is that something (my guess is Salt) does not restart Kubelet automatically if only some commandline flags are changed. At least this is what happened for me.